docs/src/pages/components/selects/CustomizedSelects.tsx

```diff
@@ -20,7 +20,6 @@ const BootstrapInput = withStyles((theme: Theme) =>
       backgroundColor: theme.palette.background.paper,
       border: '1px solid #ced4da',
       fontSize: 16,
-      width: 'auto',
```

Comment:
@oliviertassinari If you don't mind could you explain me why you removed this? The problem was with the padding right?

Reply:
It's a layout issue. We need the select element to be full-width while leaving more space on the right for the arrow.

Reply:
Okay. Thanks for the explanation.  :-)